src/types.rs

```diff
@@ -564,6 +564,17 @@ pub enum Mechanism {
     X255,
     /// Used to serialize the output of a diffie-hellman
     SharedSecret,
+
+    /// Exposes the Raw RSA encryption/decryption primitive. Be aware this is dangerous.
```

Comment:
We should also briefly explain *why* it is dangerous.

Reply:
Done